org.palladiosimulator.analyzer.slingshot.monitor/src/org/palladiosimulator/analyzer/slingshot/monitor/probes/EventBasedListProbe.java

```diff
@@ -26,27 +26,26 @@
  *
  * Beware: This probe requires a {@link MetricSetDescription}.
  *
+ * Beware: MetricSetDescriptions are define as (time, value)-pair or as (value, time)-pair. This Probe makes sure to keep thing in the correct order.
+ *
  * @author Sarah Stieß
  *
  * @param <V> The value type of the measurement.
  * @param <Q> The quantity type of the measurement.
  */
 public abstract class EventBasedListProbe<V, Q extends Quantity> extends EventBasedProbe<V, Q> {
 
+	Function<DESEvent, MeasuringValue> firstvalue;
+	Function<DESEvent, MeasuringValue> secondvalue;
```

Comment:
Do other classes in the same package need these fields? I think they should be made private just in case